mapbook-iOS/Extensions/UIColor+House.swift

```diff
@@ -0,0 +1,33 @@
+//
```

Comment:
why name this extension `House` and not `primaryBlue` ?

Reply:
It's named "House" as reference colloquially to "House Colors", meaning if we wanted to add more we could do so in the same extension.

Want me to change it?

Reply:
I approved, I was just commenting to understand the justification it's fine by me so just leave it for now